gateway/README.md

```diff
@@ -231,6 +231,20 @@ For this, you will need to specify which port to bind to.  An example config:
         "ListenAddr": "0.0.0.0:18080"
     }
 
+##### Identify and Replace variables in Span Names
+
+The SignalFx listener has the ability to ideitify and replace variables in span names and turn them into tags. It uses go's named capture groups. See more at [https://golang.org/pkg/regexp/syntax/]. 
```

Comment:
* Typo in "identify".

I don't think users know or care about Go named capture groups. Just say "It uses Regex-based replacement rules; see the [syntax here](https://golang.org/pkg/regexp/syntax/)."